docker-compose.yaml

```diff
@@ -43,26 +43,50 @@ services:
     ports:
       - "6379:6379"
 
+  local-dev-frontend:
+    user: ${DOCKER_USER:-root}
+    container_name: quay-local-dev-frontend
+    build:
+      context: .
+      target: build-python
+    image: quay-build:latest
+    init: true
+    volumes:
+      - ".:/quay-registry"
+    working_dir: /quay-registry
+    environment:
+      npm_config_cache: /quay-registry/.npm
+    entrypoint:
+      - sh
+      - -c
+      - PYTHONPATH=$$QUAYPATH python3 -m external_libraries && npm run watch
```

Comment:
@flavianmissi do we need `PYTHONPATH=$$QUAYPATH python3 -m external_libraries` here or can I remove it? We do have it right now: https://github.com/quay/quay/blob/50d2a82783da906a3a68064fe0b9ab10e956f550/local-dev/init/01_local-dev-dependencies.sh#L16

Reply:
we're storing the files that script downloads in git, so no need to run it here